server/athenian/api/models/state/models.py

```diff
@@ -12,43 +12,60 @@
 Base = create_base()
 
 
-class RepositorySet(Base):
+def create_collection_mixin(name: str):
+    """Create the collections mixin according to the required column name."""
```

Comment:
This also shouldn't be a problem 👍

Reply:
But since the fields name are dynamic I'd need to assigned them with `setattr` would that fit your needs? I don't know PyCharm intellisense and how it works.

Reply:
It does not handle `setattr`, I have checked :( Anyways, it should not be as clueless as before.